deployment/v3/external/landing-page/install.sh

```
@@ -69,7 +132,9 @@ function landing_page() {
   --set landing.healthservices=$HEALTHSERVICES \
   --set landing.injiweb=$INJIWEB \
   --set landing.injiverify=$INJIVERIFY \
-  --set istio.host=$DOMAIN
+  --set istio.host=$DOMAIN \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)